osu.Game/Tests/Platform/TestStorage.cs

```diff
@@ -7,13 +7,11 @@ namespace osu.Game.Tests.Platform
 {
     public class TestStorage : DesktopStorage
     {
-        public TestStorage(string baseName) : base(baseName, null)
+        public TestStorage(string baseName)
+            : base(baseName, null)
         {
         }
 
-        public override string GetDatabaseConnectionString(string name)
-        {
-            return "DataSource=:memory:";
-        }
+        public override string GetDatabaseConnectionString(string name) => "Data Source=" + GetUsablePathFor($"{name}.db", true);
```

Comment:
At this point "TestStorage" serves no purpose anymore. This looks to be the same thing that `DesktopStorage` does.

Is there a reason you kept it?

Reply:
good point

Reply:
Was it intended to go away from an in-memory database?

Reply:
yes, that's the point of this branch (in-memory cannot handle multiple contexts; it creates a fresh database per context).